index.html

```diff
@@ -594,52 +596,57 @@
 
 Note that the default value of **`subsurface_radius_scale`** is set at $(1, 0.5, 0.25)$ to approximate the effect of Rayleigh scattering. If we roughly approximate the wavelength corresponding to each of the RGB channels as $\lambda/\mathrm{nm} \approx 650, 550, 450$ and assume the radii scale like the reciprocal of the extinction, then since in Rayleigh scattering the extinction scales like $\lambda^{-4}$ for light of wavelength $\lambda$ the expected relative magnitudes of the radii are $(1,  (550/650)^4, (450/650)^4) \approx (1, 0.5, 0.25)$, hence the chosen default. This provides a slightly more realistic default look for the subsurface than resulting from gray radii.
 
 !!! Note: Future work
-   The particular equations for $S(\mathbf{A})$ and $\boldsymbol{\alpha}(\mathbf{A})$ presented here make certain approximations (e.g. assumptions about the dielectric interface IOR). Ideally the mapping would be generalized to an arbitrary dielectric interface.
-
    Since the scattered ray may emerge from a different point on the surface than the input point, the material will in general differ there. Thus on transmitting back out, it will encounter different layer properties. In principle this should be taken into account, but how to model subsurface scattering in the presence of varying subsurface properties is an open question.
 
 
 Subsurface params             | Label        | Type      | Range           | Norm       | Default              | Description
 ------------------------------|--------------|-----------|:---------------:|:----------:|:--------------------:|----------------------------------------------
 **`subsurface_weight`**       | Weight       | `float`   | $ [0, 1]      $ |            | $ 0                $ | Mix weight between subsurface and diffuse slabs
-**`subsurface_color`**        | Color        | `color3`  | $ [0, 1]      $ |            | $ (0.8, 0.8, 0.8)  $ | Subsurface color, i.e. the multi-scatter albedo of $V^\infty_\mathrm{subsurface}$
-**`subsurface_radius`**       | Radius       | `float`   | $ [0, \infty) $ | $ [0, 1] $ | $ 1                $ | Length scale of diffusion along the surface
-**`subsurface_radius_scale`** | Radius scale | `vector3` | $ [0, 1]^3    $ |            | $ (1.0, 0.5, 0.25) $ | RGB multiplier to **`subsurface_radius`**, giving the per-channel diffusion length
+**`subsurface_color`**        | Color        | `color3`  | $ [0, 1]      $ |            | $ (0.8, 0.8, 0.8)  $ | Subsurface color, controls the observed reflection color of $V^\infty_\mathrm{subsurface}$
+**`subsurface_radius`**       | Radius       | `float`   | $ [0, \infty) $ | $ [0, 1] $ | $ 1                $ | Length scale of MFP
+**`subsurface_radius_scale`** | Radius scale | `vector3` | $ [0, 1]^3    $ |            | $ (1.0, 0.5, 0.25) $ | RGB multiplier to **`subsurface_radius`**, giving the per-channel MFPs
```

Comment:
> (1.0, 0.5, 0.25)

Tangential: Just making sure we're going ahead with this default value as we no longer have any color inversion to compensate for.

(Incidentally, I'm going to miss the "Rayleigh scattering" parameter of the ASM, which let you achieve this type of appearance in a physically based way with a simple slide, without requiring a color to be manually entered. I suppose implementations could provide higher-level controls or presets if they want to use that concept.)

Reply:
The reasoning for this value is:

>Note that the default value of subsurface_radius_scale is set at (1,0.5,0.25)
 to approximate the effect of Rayleigh scattering. If we roughly approximate the wavelength corresponding to each of the RGB channels as $\lambda$/nm≈650,550,450 and assume the radii scale like the reciprocal of the extinction, then since in Rayleigh scattering the extinction scales like $\lambda^{-4}$ for light of wavelength $\lambda$ the expected relative magnitudes of the radii are $(1,(550/650)^4, (450/650)^4) \approx (1,0.5,0.25)$, hence the chosen default. This provides a slightly more realistic default look for the subsurface than resulting from gray radii.

Does it sound plausible to you? I noticed that the saturation of the resulting color does vary quite a bit depending on how dense the subsurface medium is.

Reply:
>(Incidentally, I'm going to miss the "Rayleigh scattering" parameter of the ASM, which let you achieve this type of appearance in a physically based way with a simple slider, without requiring a color to be manually entered. I suppose implementations could provide higher-level controls or presets if they want to use that concept.)

Yeah, it would be good to know what the logic for this higher level node should be to replicate the ASM control. We could then provide it e.g. in MaterialX.